public/app/features/provisioning/ConfigForm.tsx

```diff
@@ -153,13 +152,25 @@ export function ConfigForm({ data }: ConfigFormProps) {
             />
           </Field>
           <TokenPermissionsInfo />
-          <Field label={'Repository owner'} error={errors?.owner?.message} invalid={!!errors?.owner}>
-            <Input {...register('owner', { required: 'This field is required.' })} placeholder={'test'} />
-          </Field>
-          <Field label={'Repository name'} error={errors?.repository?.message} invalid={!!errors?.repository}>
-            <Input {...register('repository', { required: 'This field is required.' })} placeholder={'example'} />
+          <Field
+            label={'Repository URL'}
+            error={errors?.url?.message}
+            invalid={!!errors?.url}
+            description={'Enter the GitHub repository URL or owner/repository name'}
```

Comment:
owner/repository will not work in the backend, right?

Reply:
Lets remove support for that -- better to have the full scheme anyway